lib_nbgl/include/nbgl_use_case.h

```diff
@@ -137,10 +137,67 @@ typedef struct {
     uint8_t nbContents;  ///< number of contents
 } nbgl_genericContents_t;
 
+/**
+ * @brief The different types of operation to review
+ *
```

Comment:
Add a text that explains when to use what

Reply:
transaction => modifies blockchain content
message => does not modify blockchain content
operation => je sais plus

Reply:
More like:
transaction => coin/token transaction, meaning coin/token migrating from an account to another
message => message signing not modify blockchain content
operation => everything else.

Reply:
Fine for me